src/object.rs

```diff
@@ -673,33 +675,32 @@ impl<T: IsA<Object> + SetValue> ObjectExt for T {
         let property_name = property_name.into();
         let property_value = value.to_value();
 
-        unsafe {
-            let object = self.to_glib_none().0;
-            let klass = (*object).g_type_instance.g_class as *mut gobject_ffi::GObjectClass;
-            let pspec = gobject_ffi::g_object_class_find_property(klass, property_name.to_glib_none().0);
-            if pspec.is_null() {
+        let pspec = match self.find_property(property_name) {
+            Some(pspec) => pspec,
+            None => {
                 return Err(BoolError("property not found"));
-            }
+            },
```

Comment:
comma unneeded

Reply:
done

Reply:
Strange, I still see it, and in other `match self.find_property` down too.
Seems you forgot to push 😉

Reply:
I indeed forgot to push it, yes :) Thanks